--- CircleProgressView1.swift	2016-07-29 17:07:07.000000000 +0300
+++ CircleProgressView.swift	2016-07-29 17:15:36.000000000 +0300
@@ -131,7 +131,7 @@
         progressPath.addLineToPoint(CGPointMake(progressRect.midX, progressRect.midY))
         progressPath.closePath()
         
-        CGContextSaveGState(context)
+        CGContextSaveGState(context!)
         
         progressPath.addClip()
         
@@ -142,7 +142,7 @@
             circlePath.fill()
         }
         
-        CGContextRestoreGState(context)
+        CGContextRestoreGState(context!)
         
         // center Drawing
         let centerPath = UIBezierPath(ovalInRect: CGRectMake(innerRect.minX + trackWidth, innerRect.minY + trackWidth, CGRectGetWidth(innerRect) - (2 * trackWidth), CGRectGetHeight(innerRect) - (2 * trackWidth)))
@@ -150,10 +150,10 @@
         centerPath.fill()
         
         if let centerImage = centerImage {
-            CGContextSaveGState(context)
+            CGContextSaveGState(context!)
             centerPath.addClip()
             centerImage.drawInRect(rect)
-            CGContextRestoreGState(context)
+            CGContextRestoreGState(context!)
         } else {
             let layer = CAShapeLayer()
             layer.path = centerPath.CGPath
